modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -0,0 +1,84 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distereoibution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#ifndef OPENCV_GAPI_STEREO_HPP
+#define OPENCV_GAPI_STEREO_HPP
+
+#include <opencv2/calib3d.hpp>
+
+#include <opencv2/gapi/gmat.hpp>
+#include <opencv2/gapi/gscalar.hpp>
+#include <opencv2/gapi/gkernel.hpp>
+
+
+namespace cv {
+
+namespace gapi {
+
+enum class StereoOutputFormat {
+    DISPARITY,
+    DEPTH
+};
+
+namespace calib3d {
+
+struct StereoSetup {
```

Comment:
Please move this out of `calib3d` namespace.

Reply:
Please also see how the initialization parameters are declared and documented here: https://github.com/opencv/opencv/blob/master/modules/gapi/include/opencv2/gapi/video.hpp#L22
Suggest to declare and document them the same way.

Reply:
Done I hope...